src/Stage.js

```diff
@@ -61,11 +61,14 @@ export function createStageFunction() {
     return canvas;
   }
 
+  Stage.propTypes = propTypes;
+
   return Stage;
 }
 
 export function createStageClass() {
   class Stage extends React.Component {
+    static propTypes = propTypes;
```

Comment:
Is this necessary? I would like to avoid using syntax, especially that few lines above the function component is using conventional `Stage.propTypes = propTypes;` syntax

Reply:
Yeah, sorry, meant to leave a comment with this too. The proposal seems a little weird to me too. I added this static property and the dependency to get rid of some errors in the linter (only once it was pushed here though, locally it was fine), all of the stage props were getting this error: `<prop> is missing in props validation`.

I'll revert it back.

Reply:
I was able to get the eslint issues locally, all when accessing props in the class component:

```
  70:15  error  'children' is missing in props validation  react/prop-types
  70:25  error  'height' is missing in props validation    react/prop-types
  70:33  error  'options' is missing in props validation   react/prop-types
  70:42  error  'width' is missing in props validation     react/prop-types
  80:15  error  'children' is missing in props validation  react/prop-types
  80:25  error  'height' is missing in props validation    react/prop-types
  80:33  error  'options' is missing in props validation   react/prop-types
  80:42  error  'width' is missing in props validation     react/prop-types
  81:15  error  'options' is missing in props validation   react/prop-types
  92:15  error  'options' is missing in props validation   react/prop-types
```

Do you know what setting could be causing those?

Reply:
Does it happen also when you do `Stage.propTypes = propTypes` when you define `Stage` in either `createStageClass` or `createStageFunction` instead of at the end of the file?

Reply:
I got it to pass by assigning `Stage.propTypes` in both of the `create`  functions instead of at the end.